types/index.d.ts

```diff
@@ -1,8 +1,7 @@
-// TypeScript Version: 3.4
+// TypeScript Version: 4.0
 
 import {Node} from 'unist'
-import {VFile, VFileContents, VFileOptions, VFileCompatible} from 'vfile'
-import vfile = require('vfile')
```

Comment:
removed unused imports

Reply:
Maybe we should add this to `tsconfig.json` to enforce this.

```json
    "noUnusedLocals": true,
```

This could be merged even before this PR gets merged or TypeScript 4 gets released.

Reply:
resolved in #94